cmd/network-tracer/tracer.go

```diff
@@ -63,8 +64,22 @@ func (nt *NetworkTracer) Run() {
 	http.HandleFunc("/status", func(w http.ResponseWriter, req *http.Request) {})
 
 	http.HandleFunc("/connections", func(w http.ResponseWriter, req *http.Request) {
-		cs, err := nt.tracer.GetActiveConnections()
 
+		// We require to send a client_id to the network state
+		rawCID := req.URL.Query().Get("client_id")
+		clientID := ebpf.DEBUGCLIENT
+
+		if rawCID != "" {
```

Comment:
`if rawCID := req.URL.Query().Get("client_id"); rawCID != ""`

^ since its not being used outside this scope